frontend/types/src/components/Examination/ExaminationForm.vue.d.ts

```
@@ -0,0 +1,141 @@
+            default: boolean;
+        };
+    }>> & Readonly<{
+        "onUpdate:modelValue"?: ((...args: any[]) => any) | undefined;
+        "onUpdate:tempValue"?: ((...args: any[]) => any) | undefined;
+    }>, {
+        options: import("./ClassificationCard.vue").Option[];
+        modelValue: number[];
+        tempValue: number;
+        compact: boolean;
+        singleSelect: boolean;
+    }, {}, {}, {}, string, import("vue").ComponentProvideOptions, true, {}, any>;
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)